### ISSUE TITLE
Suggestion 

### ISSUE THREAD

**Issue by HellaMixel** [OnlyNoobDied/GadgetsMenu#298]
So for my Network, as a way of rewarding players, I will be giving each winner of the minigames MysteryDust. However, I want players to do something more than just buying something from the gadgetsmenu. Something I think would be awesome is a seperate GUI or wven a new plufin. This plugin features a way of gathering information from GadgetsMenu to a new GUI. This GUI will display the players current MysteryDust. Inside the GUI can have things you can buy. Fly, crates, or even kits and ranks  to help further follow EULA. I think this would be a cool feature/ new plugin. It would help being able to add custom things in the GUI and custom commands, and a custom amount of however many MysteryDust you need in order to purchase from that gui shop.

**Comment by maxlehot1234** [comment]
I think this can be done using an API. I am 99% sure that a dev can create something to get mysterydust of players using the actual API ;)

**Comment by HellaMixel** [comment]
Yes, but the shop thing is a different story. It was just an idea I had in mind.

**Comment by OnlyNoobDied** [comment]
Good idea.
